### DISCUSSION TITLE
Is the "Car Battery Level" the State of Health (SOH)? (Hyundai IONIQ 2020)

### DISCUSSION THREAD

**Discussion by andreas-bulling** [Hyundai-Kia-Connect/hyundai_kia_connect_api#452] [Q&A]
Hi,

I posted this question in the discussion forum of the kia_uno forum but thought it'd probably be a better fit here given that this is about the low-level API.

I have a question concerning the "Car Battery Level" reported in the integration: Is this the State of Health (SOH) that one can find on the battery test report generated at the Hyundai dealership (Hyundai IONIQ from 2020)?

What makes me think it might be:

1) when we got the car the battery test report showed 92% - exactly the same number reported as the "Car Battery Level" (coincidence?!)
2) the car is supposed to charge the 12v battery frequently and we can also see the blue light blinking to indicate this process. However, the "Car Battery Level" never goes up to - it actually went even further down to 85%. So this value doesn't seem to indicate the charging level of the 12v battery.

![image](https://github.com/Hyundai-Kia-Connect/hyundai_kia_connect_api/assets/377856/b24e3710-7773-472b-8274-46823deed359)

**Comment by fuatakgun** [comment]
We are bunch of enthusiastic people and working on reverse engineering the actual app traffic but we have no information what they do mean, only guessing or estimating :-)

I wish Kia/Hyundai have created an open api with good documentation so we could find about all these.
